### PR TITLE
Added s3cmd to the packages

### PR DESCRIPTION
**Pull request by yaso195**
This package is require to create apps to test our s3 gateway.

@variadico @earlruby @zquestz

### CODE REVIEW

File: packages/s3cmd-1.6.1.conf [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+environment { "PATH": "/opt/apcera/s3cmd-1.6.1/bin:$PATH" }
+
+build (
+      export INSTALLPATH=/opt/apcera
+      export PREFIX=/opt/apcera/s3cmd-1.6.1
+      sudo mkdir -p ${INSTALLPATH}
+      tar -zxf s3cmd-1.6.1.tar.gz -C $INSTALLPATH
+      cd ${INSTALLPATH}/s3cmd-1.6.1/
+      sudo -E bash -c "PATH=$PATH PYTHONHOME=$PYTHONHOME PYTHONPATH=$PYTHONPATH && python setup.py install"
+)
```
Comment (variadico) [inline packages/s3cmd-1.6.1.conf]: nit: missing newline

**Comment by variadico** [comment]
LGTM

**Comment by earlruby** [comment]
LGTM

File: packages/s3cmd-1.6.1.conf (hunk 1 above)
Comment (yaso195) [inline packages/s3cmd-1.6.1.conf]: Done

**Pull request merged**